src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'version': version,
+    'wheels': wheels,
```

Comment:
use simplified syntax
```suggestion
    name,
    version,
    wheels,
```